recipes/default.rb

```diff
@@ -1147,6 +1147,13 @@
   action :create
 end
 
+directory node['hopsworks']['data_volume']['staging_dir'] + "/connectors"  do
```

Comment:
We need to handle upgrades also.

Reply:
reverted change and included in hopsworks backend